python/runtime/cudaq/distributed/mpi_comm_impl.cpp

```diff
@@ -0,0 +1,609 @@
+/****************************************************************-*- C++ -*-****
```

Comment:
```suggestion
/*******************************************************************************
```
I believe @schweitzpgi prefers that .cpp files don't have the special line in this preamble, presumably because the tools can automatically tell it is a C++ file without any help.